CHANGELOG.md

```diff
@@ -20,6 +20,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Support the [madMAx plotter](https://github.com/madMAx43v3r/chia-plotter).
   See the [configuration wiki page](https://github.com/ericaltendorf/plotman/wiki/Configuration#2-v05) for help setting it up.
   ([#797](https://github.com/ericaltendorf/plotman/pull/797))
+- Added argument `-f / --force` to `plotman kill` to skip confirmation before killing the job
+
+### Fixed
+- `plotman kill` doesn't leave any temporary files behind anymore
```

Comment:
Let's put the fixed section first like was done below in v0.4.  Also, for both entries add a link line.

```markdown
  ([#801](https://github.com/ericaltendorf/plotman/pull/801))
```